src/utilities/cnpj/index.ts

```diff
@@ -27,12 +27,15 @@ export const FIRST_CHECK_DIGIT_WEIGHTS = [5, 4, 3, 2, 9, 8, 7, 6, 5, 4, 3, 2];
 
 export const SECOND_CHECK_DIGIT_WEIGHTS = [6, ...FIRST_CHECK_DIGIT_WEIGHTS];
 
+const VALID_CHARS_CNPJ = '0123456789ABCDFGHIJKLMNPQRSVWXYZ';
+
 export interface FormatCnpjOptions {
   pad?: boolean;
 }
```

Comment:
Eu nao sei um bom nome pra isso, mas pra evitar qualquer breaking change, acredito que de pra utilizar os options pra passar uma flag dizendo qual versao sera utilizada. A gente pode setar a versao antiga como a default, assim a gente nao quebra nada. Oq tu acha?

```suggestion
export interface FormatCnpjOptions {
  pad?: boolean;
  version?: "1" | "2"
}
```

Reply:
Nesse caso, a gente teria que atualizar a doc tambem pra refletir isso

Reply:
feito os ajustes pedidos, e atualizado doc.